index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+
+: seekable-time-ranges
+:: The time ranges for which media is seekable (ranges the controller or the
+    user on the receiver may seek to).   See
+    [HtmlMediaElement.seekable](https://html.spec.whatwg.org/multipage/media.html#dom-media-seekable).
+
+: position
+:: The playback position.  See
+    [HtmlMediaElement's official playback
+    position](https://html.spec.whatwg.org/multipage/media.html#official-playback-position)
+    and
+    [HtmlMediaElement.currentTime](https://html.spec.whatwg.org/multipage/media.html#dom-media-currenttime).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be 0.
+
+: playbackRate
+:: The current rate of playback on a scale where 1.0 is "normal speed".  See
+    [HtmlMediaElement.playbackRate](https://html.spec.whatwg.org/multipage/media.html#dom-media-playbackrate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be 1.0.
+
+: paused
+:: Whether media is paused or not.  See
+    [HtmlMediaElement.paused](https://html.spec.whatwg.org/multipage/media.html#dom-media-paused).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be false.
+
+: seeking
+:: Whether the receiver is seeking or not.  See
+    [HtmlMediaElement.seeking](https://html.spec.whatwg.org/multipage/media.html#dom-media-seeking).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be false.
+
+: stalled
+:: If true, media is not playing because not enough media is loaded, and false otherwise.  See
+    [HtmlMediaElement.stalled](https://html.spec.whatwg.org/multipage/media.html#event-media-stalled).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be false.
+
+: ended
+:: Whether media has reached the end or not.  See
+    [HtmlMediaElement.ended](https://html.spec.whatwg.org/multipage/media.html#dom-media-ended).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be false.
+
+: volume
+:: The current volume of playback on a scale of 0.0 to 1.0.  See
+    [HtmlMediaElement.volume](https://html.spec.whatwg.org/multipage/media.html#dom-media-volume).
+
+: muted
+:: True if audio is muted (overriding the volume value) and false otherwise.
+    See
+    [HtmlMediaElement.muted](https://html.spec.whatwg.org/multipage/media.html#dom-media-muted).
+
+: resolution
+:: The "intrinsic width" and "intrinsic width" of the video.  See
+    [HtmlMediaElement.videoWidth](https://html.spec.whatwg.org/multipage/media.html#dom-media-videowidth)
+    and
+    [HtmlMediaElement.videoHeight](https://html.spec.whatwg.org/multipage/media.html#dom-media-videoheight).
+
+: audio-tracks
+:: The audio tracks available, which can individually enabled or disabled.  See
```

Comment:
Grammar nit: here and below, the syntax "The <foo> available" is usually phrased as "The available <foo>."

Consider:
> :: The available audio tracks. Each track can be individually enabled or disabled.

Reply:
Done